### PR TITLE
Issue#3

### PR DESCRIPTION
**Pull request by robertb98**

### CODE REVIEW

File: frontend/src/components/forms/RegisterForm.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  },
+}));
+
+const RegisterForm = () => {
+  const classes = useStyles();
+  const { register, handleSubmit } = useForm();
+
+  const onSubmit = (data) => {
+    if (data.password1 !== data.password2) {
+      console.log("They need to be the same");
```
Comment (robertmusil) [inline frontend/src/components/forms/RegisterForm.js:37]: Why are you displaying length errors "normally", but here you're console logging them instead?

File: frontend/src/store.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import { configureStore, getDefaultMiddleware } from "@reduxjs/toolkit";
+import rootReducer from "./api/";
+import { persistStore, persistReducer } from "redux-persist";
+import storage from "redux-persist/lib/storage";
+
+const persistConfig = {
+  key: "root",
+  storage,
+  whitelist: ["user"],
```
Comment (robertmusil) [inline frontend/src/store.js:9]: can you explain what does it do?

File: frontend/src/components/forms/RegisterForm.js (hunk 1 above)
Comment (robertb98) [reply frontend/src/components/forms/RegisterForm.js:37]: I tested few things and forgot to delete this. For form errors I will use https://github.com/react-hook-form/error-message or https://www.npmjs.com/package/react-alert. I haven't decided yet.

File: frontend/src/store.js (hunk 2 above)
Comment (robertb98) [reply frontend/src/store.js:9]: I thought I would need to persist the user object between browser refresh after decoding the jwt from login. After seeing the rest of the tasks I know its unnecessary.

**Review by robertmusil — APPROVED** [review]

**Pull request merged**